provider/repodata/data/json/time_zone/exemplar_cities@1/es-AR.json

```diff
@@ -209,7 +210,7 @@
   "kgfru": "Bishkek",
   "khpnh": "Phnom Penh",
   "kicxi": "Kiritimati",
-  "kipho": "Kanton",
+  "kipho": "Enderbury",
```

Comment:
`Kanton` becoming `Enderbury` is wrong, but expected, as `Pacific/Enderbury` is listed before `Pacific/Kanton`. This problem is tracked in https://unicode-org.atlassian.net/browse/CLDR-14453. `en` works around it by having removed the display name for `Pacific/Enderbury`, so that `Pacific/Kanton` is used.